LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2019 Metrological
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Should it not have a name here instead of "[yyyy] [name of copyright owner]"?

Reply:
As per the standard, it is supposed to have a generic text in LICENSE file

Reply:
No it should have the name of the company and the year, we're suppose to fill that in :)

Reply:
I asked Lakshmipriya to do this so the file is a generic Apache license and any contributor to the code base can be credited with one copy of the file in LICENSE.  Also, going forward, code here will have RDK Apache header, not Metrological.  You are right in that whereever the header is deployed, it will have a real year and company.

Reply:
Looking at other open source repo's I do see an actual name and year.
[link](https://github.com/GoogleChrome/chrome-launcher/blob/master/LICENSE#L190)

So I still believe it should contain a name and year

Reply:
Hi Wooter.  Yes, anywhere the header is actually used, it must be specific.  But in the generic license file, we want it to be generic.  This is the convention in RDK.

Reply:
Yes then please add the RDK company name and 2020 year.

Reply:
Hi Wouter.  I am following the established RDK convention used in over a hundred components.   The section to which you refer is an Appendix in the license text, not the header to be applied to every file.  It makes sense to retain a completely generic license text here so that in components where more than one body of software is apache-licensed, we do not need to include two almost identical copies of the apache license.  (Case in point: Thunder has RDK/Metrological and ARM code, both Apache licensed.  I see no point in including the license text twice.  (This is actually done in some RDK components which were prepared before my involvement, and looks very odd.)

If you are not happy with this, then we will simply delete the appendix (everything after "End of Terms and conditions").

Please let me know.

Reply:
I don't think "that's the way we always do it" is a very compelling argument. That's what bad religions are built on :)

The reason I'm pushing back on this is that we used to have the same generic [] copy text in our repositories a while back. However during a certain event in last summer OUR legal (which applies to both Metro, Comcast and RDK) gave feedback that having a generic, non reference to a company in the license is not allowed and we had to update them. And well angry lawyers really make me nervous so we updated it. I'd like to keep the angry lawyers at bay and follow their directive that we initially received. In fact, judging by your over a hundred components remark, I think you should cross check that as well (unless well you like angry lawyers).

On top of that ThunderUI is a sole Metrological developed component, it does not have RDK or ARM code in it, making it quite clear who's the licensing entity here. 

Agree we don't need this in header of every file, but we do need it in the main LICENSE file. Which is where this thread originally started.

Reply:
Hi Wouter.  I will ask RDK Management for a determination.

Reply:
Hi Wouters.  In the interests of a rapid approval, Glee has requested that this be done the Metrological way for now.  Glee will take up the question with legal.

Reply:
Appreciated! Looking forward to the updates.